RetroBar/Utilities/ToolbarDropHandler.cs

```diff
@@ -14,11 +14,21 @@ public ToolbarDropHandler(Toolbar toolbar)
             _toolbar = toolbar;
         }
 
+        public void DragEnter(IDropInfo dropInfo)
+        {
+            DragDrop.DefaultDropHandler.DragEnter(dropInfo);
+        }
+
         void IDropTarget.DragOver(IDropInfo dropInfo)
         {
             DragDrop.DefaultDropHandler.DragOver(dropInfo);
         }
 
+        public void DragLeave(IDropInfo dropInfo)
+        {
+            DragDrop.DefaultDropHandler.DragLeave(dropInfo);
+        }
+
```

Comment:
What is the reason for these additions?

Reply:
Maybe it is only needed for .NET Framework, it is there so it compiles with that version.

Check this: https://github.com/punker76/gong-wpf-dragdrop/blob/3.0.0/src/GongSolutions.WPF.DragDrop/IDropTarget.cs

Reply:
That makes sense. Maybe these should be `#if`'d?

Reply:
Yes, that's better, I'll make the changes.